test/00/t0027a.sh

```diff
@@ -35,6 +35,7 @@ cat >input.tcl <<EOF
 source assert.tcl
 proc afterMinskyStarted {} {
 uplevel #0 {
+minsky.save no
```

Comment:
Are you planning on leaving this line in?

Reply:
No, I hope that making the boolean variable multipleEquities const in minsky.h, as you have suggested, solves the save at exit warning that has caused the test suite to hang.